Makefile

```diff
@@ -9,24 +9,21 @@ endif
 
 ALL: $(CMDS)
 
-ui: ui/bindata.go
+ui: ui/.build/ui
 
 node_modules:
 	npm install
 
-$(GOPATH)/bin/houndd: ui/bindata.go $(SRCS)
+$(GOPATH)/bin/houndd: ui/.build/ui $(SRCS)
 	go install github.com/hound-search/hound/cmds/houndd
 
 $(GOPATH)/bin/hound: ui/bindata.go $(SRCS)
 	go install github.com/hound-search/hound/cmds/hound
 
-.build/bin/go-bindata:
-	GOPATH=`pwd`/.build go get github.com/go-bindata/go-bindata/...
-
-ui/bindata.go: .build/bin/go-bindata node_modules $(wildcard ui/assets/**/*)
-	rsync -r ui/assets/* .build/ui
+ui/.build/ui: node_modules $(wildcard ui/assets/**/*)
+	mkdir -p ui/.build
+	rsync -r ui/assets/* ui/.build/ui
 	npx webpack $(WEBPACK_ARGS)
```

Comment:
I think the original motivation around using bindata in the first place is so that we could ensure built static assets were present and usable immediately after `go install`, without requiring an additional build step / installing `node_modules` / etc. I'm admittedly fuzzy about whether or not Go Modules have a post-build lifecycle step, but I think that context is important for this PR.

Reply:
I have had success with [pkger](https://github.com/markbates/pkger).